src/getRandomNum.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports.getRandom = () => {
+  const randomNumber = Math.random() * (9999 - 1000) + 1000;
+
+  return Math.floor(randomNumber);
+};
```

Comment:
It will not pass the p#1 and p#4, because it must be 4 different digits, but here I can get for example number 4444,or 5544, ect.